### PR TITLE
btn and input refactoring

### PR DESCRIPTION
**Pull request by Maxim-Mal**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -146,26 +146,28 @@
...
-        viewBox="0 0 40 29"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          d="M39.6554 0.910205L20.4101 15.1408C20.1718 15.3172 19.8451 15.3172 19.5978 15.1497L0.149944 1.2896C0.0529528 1.52791 0 1.78408 0 2.0575V26.8905C0 28.0294 0.92711 28.9562 2.0657 28.9562H37.9339C39.0728 28.9562 39.9996 28.0291 39.9996 26.8905V2.06617C40.0085 1.6422 39.8761 1.2361 39.6553 0.909692L39.6554 0.910205ZM1.06783 0.256838L19.9955 13.7371L38.4723 0.0708171C38.3045 0.0267876 38.1191 0.000308502 37.9337 0.000308502L2.07345 0C1.71167 0 1.36744 0.0883679 1.0672 0.256173L1.06783 0.256838Z"
+    <div class="subscribe">
+      <div class="input-field">
+        <input
+          type="text"
+          class="input-field input-field__input"
```
Comment (Malchenko-Yehor) [inline public/index.html:153]: Why we have double input-filed class, it should be only on parent div
```suggestion
          class="input-field__input"
```

**Review by Malchenko-Yehor — COMMENTED** [review]
Don't set fixed width on button--link

File: src/styles/components/_btn.scss [hunk 2]
```diff
@@ -53,8 +53,11 @@
   &--lang {
     transition: all 0.3s ease;
   }
+  &--subscribe {
+    margin-left: 20px;
```
Comment (Malchenko-Yehor) [inline src/styles/components/_btn.scss:57]: Remove subscribe modificator and use --link modificator, to make indent in subscribe blok, create subscribe component and apply margin to button there

**Review by Malchenko-Yehor — COMMENTED** [review]
Remove variables import in all components

File: src/styles/components/_input.scss [hunk 3]
```diff
@@ -8,6 +8,7 @@
   &__input {
     height: 45px;
     width: 250px;
```
Comment (Malchenko-Yehor) [inline src/styles/components/_input.scss:10]: Don't set fixed width on input

File: src/styles/components/_input.scss [hunk 4]
```diff
@@ -41,6 +42,7 @@
     &--letter {
       fill: $btn-text-color;
     }
+
     &--path {
       stroke: #c4c8d1;
```
Comment (Malchenko-Yehor) [inline src/styles/components/_input.scss:47]: Use variable instead

File: public/index.html (hunk 1 above)
Comment (Maxim-Mal) [reply public/index.html:153]: done

File: src/styles/components/_btn.scss (hunk 2 above)
Comment (Maxim-Mal) [reply src/styles/components/_btn.scss:57]: Done

File: src/styles/components/_input.scss (hunk 3 above)
Comment (Maxim-Mal) [reply src/styles/components/_input.scss:10]: Done

File: src/styles/components/_input.scss (hunk 4 above)
Comment (Maxim-Mal) [reply src/styles/components/_input.scss:47]: Done

File: src/styles/components/_navigation-panel.scss [hunk 5]
```diff
@@ -2,7 +2,8 @@
 @import "input";
 
 .nav-panel {
+  padding: 0 100px 0 100px;
```
Comment (Malchenko-Yehor) [inline src/styles/components/_navigation-panel.scss:5]: Better use shorthand padding expressing in this case
```suggestion
  padding: 0 100px;
```

**Pull request merged**